docs/driverapi.md

```diff
@@ -0,0 +1,398 @@
+# Driver API
+
+Below is a pictorial representation of the `Driver API`:
+
+![driverapi.png](imgs/driverapi.png)
+
+## Serialization
+
+The Driver API recommends to use the `protobuf` protocol to serialize public parameters, token requests.
+The relative protobuf messages are [`here`](https://github.com/hyperledger-labs/fabric-token-sdk/blob/2118c1535ebaaed2ecc293b0c3f66aa71eeafe21/token/driver/protos/).
+This guarantees retro and forward compatibility.
+
+The message for the public parameters carries:
+- A token driver identifier to be able to understand which driver generated these parameters.
+- The lower-lever representation of the public parameters. Each driver must decide how to encode these bytes.
+
+The message for the token requests consists of:
```

Comment:
The term "token requests" is first defined here. Is there an equivalent term "token responses"?